include/netlink-private/types.h

```diff
@@ -254,6 +255,7 @@ struct rtnl_neigh
 	uint8_t		n_type;
 	struct nl_addr *n_lladdr;
 	struct nl_addr *n_dst;
+	uint32_t	n_nhid;
```

Comment:
this needs to go the the previous commit (2 of 4). Otherwise, that commit does not compile.

Reply:
done